crates/common/fmt/src/ui.rs

```diff
@@ -338,7 +338,7 @@ to                   {}
 transactionIndex     {}
 type                 {}
 value                {}
-yParity              {}{}",
+yParity              {}",
```

Comment:
We should still format additional fields, I think we can just impl `UIfmt` for `WithOtherFields<Transaction>`?